bridge_adaptivity/module/mixins.py

```diff
@@ -43,6 +46,39 @@ def dispatch(self, request, *args, **kwargs):
 
 
 class GroupEditFormMixin(object):
+    form_class = GroupForm
+    prefix = 'group'
```

Comment:
I don't think so... because this mixin is only about editing\updatin\creating CollectionGroup's it will create form with prefix ```group``` to separate forms.

Reply:
already discussed it...